content/en/docs/developer_guide/console/_index.md

```diff
@@ -135,6 +135,12 @@ Before installing community plugins, you should confirm that you agree that you
 
 To create a plugin, select the **Create plugin** link. You are directed to [{{% variables/apibuilder_prod_name %}} SDK](/docs/developer_guide/sdk) documentation to guide you through the process.
 
+### Updates tab
+
+The **Updates** tab shows updates and upgrades for installed components. You should always keep your components up to date for the latest features, bug fixes, and security updates.
```

Comment:
We said in the ticket to mention any limitations here. We should consider saying that this feature will only work (for now) on Node.js '^12.13.0 || ^14.15.0 || >=16.0.0';

Reply:
You did cover this yesterday. Closing.